backend/src/test/java/org/cryptomator/hub/api/VaultResourceTest.java

```diff
@@ -162,10 +162,10 @@ public void testUnlockArchived2() {
 		}
 
 		@Test
-		@DisplayName("GET /vaults/7E57C0DE-0000-4000-8000-00010000AAAA/access-token returns 403 for archived vaults with evenIfArchived set to true")
+		@DisplayName("GET /vaults/7E57C0DE-0000-4000-8000-00010000AAAA/access-token returns 200 for archived vaults with evenIfArchived set to true")
 		public void testUnlockArchived3() throws SQLException {
 			when().get("/vaults/{vaultId}/access-token?evenIfArchived=true", "7E57C0DE-0000-4000-8000-00010000AAAA")
-					.then().statusCode(403);
+					.then().statusCode(200);
 		}
```

Comment:
Why did the old test succeed? 😭

Reply:
Quiet exactly for the same reason described in the bug report: In the [testGrantAccessArchived](https://github.com/cryptomator/hub/pull/270/files#diff-26d7286dcd2fe62fc0616f129b85cc729ce7a5532a3c85c64d2385dd6c717942R384), the access-token wasn't added because [this vault is archived](https://github.com/cryptomator/hub/blob/develop/backend/src/test/resources/org/cryptomator/hub/flyway/V9999__Test_Data.sql#L42-L45) and then the `GET /vaults/{vaultId}/access-token?evenIfArchived=true` resource throws a [ForbiddenException](https://github.com/cryptomator/hub/blob/develop/backend/src/main/java/org/cryptomator/hub/api/VaultResource.java#L327) (403).